### PR TITLE
Milestone 3: Awesome books - with ES6

### PR DESCRIPTION
**Pull request by odhiambo-ed**
In this milestone, I did the following:

    - Added the entry point for JavaScript code called index.js
    - Added index.css file for styles.
    - Added the main index.html file and linked index.js and index.css files there.
    - Added a directory called modules.
    - Re-created Awesome book app using ES6.
    - Displayed current date using Luxon.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# .gitignore
+node_modules/
```
Comment (ShoiraTa) [inline .gitignore:2]: - [ ] Please make sure to delete the node_modules from the repository. Here is what you can do:
- Delete node modules, 
- Push changes 
- Run `npm install `

File: modules/deleteBookRecord.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+/* eslint-disable import/prefer-default-export */
```
Comment (ShoiraTa) [inline modules/deleteBookRecord.js]: - [ ] Please avoid making custom changes in disabling `eslint`.
If you have one function to export, then at the end of document you can write 

`export default myFunction`
then import it:
`import myFunction from './modules/index.js'`

File: modules/displayTime.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+/* eslint-disable import/prefer-default-export */
```
Comment (ShoiraTa) [inline modules/displayTime.js]: - [ ] Please avoid making custom changes in disabling `eslint`.
If you have one function to export, then at the end of document you can write 

`export default myFunction`
then import it:
`import myFunction from './modules/index.js'`

File: modules/recordNewBook.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+/* eslint-disable import/prefer-default-export */
```
Comment (ShoiraTa) [inline modules/recordNewBook.js]: - [ ] Please avoid making custom changes in disabling `eslint`.
If you have one function to export, then at the end of document you can write 

`export default myFunction`
then import it:
`import myFunction from './modules/index.js'`

File: modules/deleteBookRecord.js [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+        && deleteButton.classList.contains('delete')
+  ) {
+    e.preventDefault();
+    const tr = deleteButton.closest('tr');
+    const isbn = tr.getAttribute('data-isbn');
+    tr.remove();
+
+    bookList.removeNewBook(isbn);
+  }
+};
```
Comment (ShoiraTa) [inline modules/deleteBookRecord.js]: - [ ] It is not possible to delete books, after page refresh all books are listed back. Please try to delete books and check your console:
Screenshot 2022-03-22 at 20 41 12

- [ ] Please make sure to address this issue 🙏🏻

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
STATUS: CHANGES REQUIRED 🟡 
----
Hi @white3d ,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

There are some issues that you still need to work on to go to the next project but you are almost there!

woohoo-youre-almost


 ## Required Changes  :recycle: 

- [ ] Please kindly update the README file by adding all the relevant sections as `getting started`, `Authors`, `License` etc. Please add the project description as well. 

- [ ] Check the comments under the review.

## Optional suggestions

- [ ] OPTIONAL: After I click on refresh button, it takes too long for page to get refreshed. I suggest to take a look at this issue 

 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Edward, 
## Congratulations 🎉 Thank you for implementing all the requirements 🙏
### Please go ahead and merge the branch!

## Happy Coding 👏👏👏
#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**